packages/app/__tests__/integration/ecart-rep/mock/user.ts

```diff
@@ -0,0 +1,41 @@
+import type { TokenInfoType } from "@services/apiClient";
+
+const FAKE_SIREN = "23274530763485";
+
+/**
+ * @link https://www.pappers.fr/entreprise/boulangerie-eden-833169014
+ */
+const BOULANGERIE_EDEN_SIREN = "833169014";
+
+export const FAKE_USER: TokenInfoType = {
+  staff: true,
+  déclarations: [],
+  email: "test@test.com",
+  ownership: [FAKE_SIREN, BOULANGERIE_EDEN_SIREN],
+};
+
+export const useUserMock = (isConnected: boolean) => ({
+  useUserStore() {
+    return {
+      setToken: () => void 0,
```

Comment:
void 0 ?

Reply:
C'est du code en pair programming avec @lsagetlethias 
Vu cet article [https://p42.ai/blog/2022-05-10/you-dont-need-void-0](https://p42.ai/blog/2022-05-10/you-dont-need-void-0) on peut remplacer par `undefined`, ça vous va @lsagetlethias @pom421 ?

Reply:
@pom421 Ce code est déjà présent sur la PR #1221 , je le corrigerai sur la PR de base.

Reply:
`void 0` ou même `void "toto"` (ou n'importe quel primitif après l'opérateur void) c'est égal à undefined. Ou plus précisément à "pas de retour" ; c'est un réflexe de code que j'ai depuis l'époque ou les bundler n'éxistait pas et où `void 0` était préféré à `undefined` car moin de caractères.

Reply:
C'est pas plus mal d'utiliser `void 0` ou `undefined`, faut juste que ce soit pas bloquant pour merger la PR car j'en ai une PR qui dépend de celle-ci